### DISCUSSION TITLE
Hostname service information missing in Wiki

### DISCUSSION THREAD

**Discussion by jonathanalevi** [Guilouz/Creality-Helper-Script-Wiki#421] [Issues]
I see that things have moved to the Wiki, however, there's no information about the hostname service or how it works there. 
I installed, but can't seem to get it working. Needs documentation. Thanks!

**Comment by Guilouz** [comment]
Hostname Service have been removed in new version because it's already enabled by default in latest Creality firmware.

**Reply by jattie-ire** [reply]
Is there a way to permanently change my printer hostname from the default assigned, when I reboot, the defaults are restored. I found the shell scrip that sets the hostname on reboot at: /etc/init.d/S01hostname and figured out that they are executed in numerical order at startup, so I renamed it to _S01hostname to avoid the renaming of the stored hostname in the hostname file and this works for now, however I would like to know if there is a better way to handle this. 

Thanks

**Reply by jattie-ire** [reply]
I figured it out, just edit or move the file name to not start with S, done same way in installer scripts.

**Comment by Brakisshuro** [comment]
So if it is supported, how do we change it via ssh?

**Reply by jattie-ire** [reply]
mv /etc/init.d/S01hostname /etc/init.d_S01hostname
